rasa/core/channels/telegram.py

```diff
@@ -177,6 +177,10 @@ def _is_location(message: Message) -> bool:
     def _is_user_message(message: Message) -> bool:
         return message.text is not None
 
+    @staticmethod
+    def _is_edited_message(message: Message) -> bool:
```

Comment:
I think `message` should be of type `Update` here - `Message` doesn't have `edited_message` attribute.

Reply:
I think you are right, I have changed it to `Update`.